carma_wm_ros2/src/CARMAWorldModel.cpp

```diff
@@ -1502,26 +1516,24 @@ namespace carma_wm
 
             auto received_state_dynamic = static_cast<lanelet::CarmaTrafficSignalState>(current_movement_event.event_state.movement_phase_state);
             
-            bool recorded = check_if_seen_before_movement_state(min_end_time_dynamic,received_state_dynamic,curr_intersection.id.id,current_movement_state.signal_group);
-            
-            if (!recorded)
-		        {
-              sim_.traffic_signal_states_[curr_intersection.id.id][current_movement_state.signal_group].push_back(std::make_pair(min_end_time_dynamic, received_state_dynamic));
-              sim_.traffic_signal_start_times_[curr_intersection.id.id][current_movement_state.signal_group].push_back(
+            //bool recorded = check_if_seen_before_movement_state(min_end_time_dynamic,received_state_dynamic,curr_intersection.id.id,current_movement_state.signal_group);
+                        
+            sim_.traffic_signal_states_[curr_intersection.id.id][current_movement_state.signal_group].push_back(std::make_pair(min_end_time_dynamic, received_state_dynamic));
+            sim_.traffic_signal_start_times_[curr_intersection.id.id][current_movement_state.signal_group].push_back(
                                 start_time_dynamic);
               
-              RCLCPP_DEBUG_STREAM(rclcpp::get_logger("carma_wm_ros2"), "intersection id: " << (int)curr_intersection.id.id << ", signal: " << (int)current_movement_state.signal_group
-                 << ", start_time: " << std::to_string(lanelet::time::toSec(start_time_dynamic))
-                 << ", end_time: " << std::to_string(lanelet::time::toSec(min_end_time_dynamic))
-                 << ", state: " << received_state_dynamic);
-                 
-              curr_light->recorded_time_stamps = sim_.traffic_signal_states_[curr_intersection.id.id][current_movement_state.signal_group];
-              curr_light->recorded_start_time_stamps  = sim_.traffic_signal_start_times_[curr_intersection.id.id][current_movement_state.signal_group];
-            }
-	        }
-        } 
+            RCLCPP_DEBUG_STREAM(rclcpp::get_logger("carma_wm_ros2"), "intersection id: " << (int)curr_intersection.id.id << ", signal: " << (int)current_movement_state.signal_group
+             << ", start_time: " << std::to_string(lanelet::time::toSec(start_time_dynamic))
+             << ", end_time: " << std::to_string(lanelet::time::toSec(min_end_time_dynamic))
+             << ", state: " << received_state_dynamic);
+          }
```

Comment:
This brace is extra. Please remove it.

Reply:
This is not extra, but a bad indentation, so fixed it